Shared/Mirror/mirrorClient.js

```diff
@@ -0,0 +1,314 @@
+"use strict";
+
+(function () { // BEGIN LOCAL SCOPE
+
+    // VARIABLES
+	
+    var _this = this;
+	var debug = false;	// If true then a spectator camera model and box entity will be rezzed
```

Comment:
use 4 spaces for indention, please check your code editor settings and set it to use 4 spaces instead of tabs.